src/features/public/Donations/components/ThankYou.tsx

```diff
@@ -113,6 +113,10 @@ function ThankYou({
 
       <div className={styles.thankyouImageContainer}>
         <div className={styles.thankyouImage}>
+          <div className={styles.thankyouImageHeader}>
+            <p>{t('donate:thankyouHeaderText')}</p>
```

Comment:
If possible please make 1 string use `\n`

Reply:
Yes we can do it that way but the **Trillion Trees** has a different style if we make it one one sentence, i'm not sure if it is possible to add the styles to it